_components/accordions.md

```diff
@@ -4,9 +4,153 @@ type: component
 title: Accordions
 ---
 
-<div class="preview">
-  <!-- Add HTML markup for example here -->
-  <img src="{{ site.baseurl }}/assets/img/static/Accordions_UI_v1.png">
+<h3>Bordered Accordion</h3>
+
+<div class="code-sample">
+  <ul id="accordion-menu">
```

Comment:
While I can understand the assumption that users would want to use an accordion more than once on a page, I'd rather leave that up to our UX team to determine. Let's go ahead and leave this as an ID for now and point out to the team that this is a good thing to test for. //cc @carodew @colinpmacarthur